### ISSUE TITLE
Pure daisy in vanilla enchantment table = bug

### ISSUE THREAD

**Issue by LuciousLisa** [Vazkii/Botania#1924]
Putting a pure daisy in a vanilla enchantment table results in a 'Botania Flower', indicating a bug.

[screenshot](http://i.imgur.com/HI1UaNd.png)

Thanks for the achievement btw. ;-)

**Comment by Vazkii** [comment]
Can't reproduce.

**Comment by LuciousLisa** [comment]
I bet I can reproduce it. What do you need to know?

**Comment by Vazkii** [comment]
https://github.com/Vazkii/Botania/blob/master/contributing.md

**Comment by LuciousLisa** [comment]
That's a great link! I was actually looking for such a link in the readme.md, but couldn't find it. Forgive me for not going through the entire repo. I did a search on the issues, checked to see if I could find ANY issues involving the word daisy (nope) and decided to report the (literally) single step needed to reproduce the issue. I even included a screenshot, so I figured I had it covered. Guess not.

Pro-tip: include that link in the readme and show some grace towards people that are actually willing to pause the game for long enough to report the issue with YOUR project. I absolutely love your mod, don't get me wrong, but closing an issue with an unhelpful one-liner does not encourage me to report another.

Anyway, I'm using the Regrowth pack. Go from there. Or not. Good luck and thanks for the mod.
